build/brew-template.rb

```diff
@@ -18,9 +19,14 @@ def install
     doc.install resource( "apidocs" )
   end
 
+  test do
```

Comment:
Not unless you run the "brew test ..." command.  In order to pass Homebrew's master formula repo builds, you're required to have tests in the Formula class.